local_rrt_detector:
  start_time: 7.5
  eta: 1.0 # how greedy the search is